.editorconfig

```diff
@@ -80,3 +80,207 @@ dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.sev
+dotnet_diagnostic.SA1309.severity = none            # Field names should not begin with underscore.
+dotnet_diagnostic.SA1310.severity = none            # Field names should not contain underscore.
+dotnet_diagnostic.SA1311.severity = none            # Static readonly fields should begin with upper-case letter.
+dotnet_diagnostic.SA1312.severity = none            # Variable '_' should begin with lower-case letter.
+dotnet_diagnostic.SA1313.severity = none            # Parameter '_' should begin with lower-case letter.
+dotnet_diagnostic.SA1314.severity = none            # Type parameter names should begin with T.
+
+# StyleCop: Maintainability Rules
+dotnet_diagnostic.SA1400.severity = none            # Access modifier should be declared.
+dotnet_diagnostic.SA1401.severity = none            # Fields should be privat.
+dotnet_diagnostic.SA1402.severity = none            # File may only contain a single type.
+dotnet_diagnostic.SA1403.severity = none            # File may only contain a single namespace.
+dotnet_diagnostic.SA1404.severity = none            # Code analysis suppression should have justification.
+dotnet_diagnostic.SA1405.severity = none            # Debug.Assert should provide message text.
+dotnet_diagnostic.SA1406.severity = none            # Debug.Fail should provide message text.
+dotnet_diagnostic.SA1407.severity = none            # Arithmetic expressions should declare precedence.
+dotnet_diagnostic.SA1408.severity = none            # Conditional expressions should declare precedence.
+dotnet_diagnostic.SA1409.severity = none            # Remove unnecessary code.
+dotnet_diagnostic.SA1410.severity = none            # Remove delegate parenthesis when possible.
+dotnet_diagnostic.SA1411.severity = none            # Attribute constructor should not use unnecessary parenthesis.
+dotnet_diagnostic.SA1412.severity = none            # Store files as UTF-8 with byte order mark.
+dotnet_diagnostic.SA1413.severity = none            # Use trailing comma in multi-line initializers.
+
+# StyleCop: Layout Rules
+dotnet_diagnostic.SA1500.severity = none            # Braces for multi-line statements should not share line.
+dotnet_diagnostic.SA1501.severity = none            # Statement should not be on a single line.
+dotnet_diagnostic.SA1502.severity = none            # Element should not be on a single line.
+dotnet_diagnostic.SA1503.severity = none            # Braces should not be omitted.
+dotnet_diagnostic.SA1504.severity = none            # All accessors should be single-line or multi-line.
+dotnet_diagnostic.SA1505.severity = none            # A closing brace should not be preceded by a blank line.
+dotnet_diagnostic.SA1506.severity = none            # Element documentation headers should not be followed by blank line.
+dotnet_diagnostic.SA1507.severity = none            # Code should not contain multiple blank lines in a row.
+dotnet_diagnostic.SA1508.severity = none            # Closing braces should not be preceded by blank line.
```

Comment:
Yes, we could suggest this one

Reply:
@jtkech 
yes we could suggest more than just this one. But, I think in this PR, we should merge it as is if agreed. Once we merge it, we can use follow up PR to enable and apply which every rules we want. this way, it's easy to clean up the project using smaller PRs with one/two rules at a time.